docs/src/tutorials/tutorial_2.md

```diff
@@ -367,18 +335,17 @@ An example resulting electrode configuration with random colors looks like this:
 
 ```julia
 ...
-        Action(
-            :electrodes,
-            (args...) ->
-                electrode.(
-                    electrode_locations,
-                    rand(indicators, length(electrode_locations)),
-                    "black",
-                    :fill,
-                    radius,
-                    electrode_names,
-                ),
+electrodes = Object(
```

Comment:
I'm actually wondering whether it makes sense to make an object for each electrode such that you can change the color of each one individually.

Reply:
I could, but I think that is getting beyond just using Javis to make something and instead using Julia to make it more realistic. Do you think this would highlight aspects of Javis more or would be for more on aesthetics?

Reply:
It's okay. It's basically a general remark for you :wink: